internal/platform/database/billing/repository.go

```
@@ -143,6 +167,276 @@ func NewRepository(db *sql.DB) *SQLRepository {
+FROM bills
+WHERE due_date < $1
+  AND status = 'pending_payment'
+  AND deleted_at IS NULL
+ORDER BY due_date ASC, id ASC
+`
+
+	rows, err := r.db.QueryContext(ctx, query, today)
+	if err != nil {
+		return nil, fmt.Errorf("list overdue scan candidates: %w", err)
+	}
+	defer rows.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `rows.Close` is not checked

(errcheck)